### PR TITLE
Update of GitHub screenshots

### LINKED ISSUE #254: Switch to "main" as default branch
This is dependent on https://github.com/coderefinery/git-intro/issues/352

If we change to "main" as default, please check all screenshots and graphics.

### PR DESCRIPTION
**Pull request by johanhellsvik**
Updated GitHub screenshots. This in order to reflect how these GitHub pages/panels are looking currently.
Note that 'master' is used as the base name in some of the repositories for which screenshots were made.
Closes #254

### CODE REVIEW

File: content/img/centralized/generate_repo.svg
Comment (dhanyapushpadas) [inline content/img/centralized/generate_repo.svg:1]: Arrow head formatting ?

File: content/img/centralized/unwatch.svg
Comment (dhanyapushpadas) [inline content/img/centralized/unwatch.svg:1]: Can we fix the arrow heads like the previous version

Comment (johanhellsvik) [reply content/img/centralized/unwatch.svg:1]: I shared another variant on Zulip. Not identical but with slightly elongated arrow head. Can we go for that one?

Comment (dhanyapushpadas) [reply content/img/centralized/unwatch.svg:1]: Yes the zulip variant is good. Thanks!

Comment (johanhellsvik) [reply content/img/centralized/unwatch.svg:1]: Thank you! Then I will revise all figures in the same style.

Comment (johanhellsvik) [reply content/img/centralized/unwatch.svg:1]: fixed in the last commit

Comment (dhanyapushpadas) [reply content/img/centralized/unwatch.svg:1]: Thanks! @johanhellsvik Can we delete the svg files or you want to keep them too?

Comment (johanhellsvik) [reply content/img/centralized/unwatch.svg:1]: Thank you! I put the svg files there intentionally and suggest we keep them.

Comment (dhanyapushpadas) [reply content/img/centralized/unwatch.svg:1]: Thanks, then I will merge it.

**Pull request merged**